### ISSUE TITLE
NTR: Manganese low in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8873]
New term request Manganese low in urine (elthat_160116173154):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000481
- label: Reduced urinary manganese level
- parents: [Abnormal urinary mineral level (HP:0025640)](https://hpo.jax.org/app/browse/term/HP:0025640)
- pmid: PMID:1577529
- def: The amount of manganese in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Manganese is a trace mineral required for metabolism, growth and tissue formation, and reproduction. It is mainly obtained through food and water, as well as through occupational exposure.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8873
- synonyms:

**Comment by kanems** [comment]
Does this definition have an error?
The preferred name is 'reduced' but the definition says 

> The amount of manganese in the urine, normalized for urine concentration, is _above the upper limit_ of normal.
Should it be _below the lower limit_?

**Comment by MickeySegal** [comment]
The label is correct and the definition is wrong.  Good pick-up.

**Comment by pnrobinson** [comment]
Fixed, thanks!
